### PR TITLE
Update constructr to version 0.13.0

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -2,7 +2,7 @@ import sbt._
 
 object Version {
   final val Scala            = "2.11.8"
-  final val ConstructrAkka   = "0.12.0"
+  final val ConstructrAkka   = "0.13.0"
   final val RaptureJsonSpray = "1.1.0"
   final val Akka             = "2.4.2"
```
Comment (gerson24) [inline project/Dependencies.scala]: Can you update akka to version 2.4.4?

File: constructr-coordination-consul/src/main/scala/com.tecsisa.constructr.coordination.consul/ConsulCoordination.scala [hunk 2]
```diff
@@ -18,12 +18,14 @@ package com.tecsisa.constructr.coordination
...
+import akka.http.scaladsl.Http
 import akka.http.scaladsl.client.RequestBuilding.{ Get, Put }
 import akka.http.scaladsl.model.MediaTypes.`application/json`
 import akka.http.scaladsl.model.StatusCodes.{ NotFound, OK }
 import akka.http.scaladsl.model.{ HttpRequest, HttpResponse, HttpEntity, ResponseEntity, RequestEntity, StatusCode, Uri }
 import akka.http.scaladsl.unmarshalling.Unmarshal
-import akka.stream.Materializer
+import akka.stream.ActorMaterializer
 import akka.stream.scaladsl.{ Sink, Source }
 import com.typesafe.config.Config
```
Comment (gerson24) [inline constructr-coordination-consul/src/main/scala/com.tecsisa.constructr.coordination.consul/ConsulCoordination.scala]: Please remove unnecessary imports :)

File: project/Dependencies.scala (hunk 1 above)
Comment (raboof) [inline project/Dependencies.scala]: done!

**Comment by raboof** [comment]
@gerson24 done!

**Comment by gerson24** [comment]
Please follow [CONTRIBUTING](https://github.com/Tecsisa/constructr-consul#contribution-policy) guide for PRs.
Could you squash these into one commit, and reference the issue #12 in the commit message?

Other than that LGTM.
@raboof thanks a lot for contributing!

**Comment by raboof** [comment]
Thanks for the merge, I'll squash #10 tomorrow!

**Pull request merged**